cookbook/routing/index.rst

```diff
@@ -9,6 +9,8 @@ Routing
     redirect_in_config
     method_parameters
     service_container_parameters
-    custom_route_loader
     redirect_trailing_slash
     extra_information
+    extending_routing
+        custom_route_loader
+        multiple_routers
```

Comment:
yep, found it :-)

but we need to solve the general question how we organize the
information, whether we compact that into one article or something else.